### DISCUSSION TITLE
Sourcesync Pro

### DISCUSSION THREAD

**Discussion by fioan89** [fioan89/sourcesync#103] [Ideas]
Greetings Sourcesync contributors. I am Ioan, the creator and maintainer of Sourcesync. I _summoned_ everyone here because I would like to gather some feedback about Sourcesync.

For the last couple of months I've been thinking about releasing a paid **pro** version of Sourcesync. No worries, Sourcesync will still be open source, free of charge, and actively maintained. There are a couple of enhancement that are recurrently requested by the community, for example:
- ability to sync directories using SSH and SCP
- ability to auto-sync saved files
- support for SSH agents and SSH Proxy Commands

The **pro** version could have all the features from the free version + all the highly requested enhancements in the past couple of years. Before going further I would like to ask everyone:
- how would you feel towards a **pro** version?
- what features/improvements would you like to see in the **pro** version
- would you buy/use the **pro** variant?
- what would be a comfortable price you would pay for the extra features? Right now Jetbrains Marketplace supports only monthly/yearly subscriptions. I know some people are not comfortable with subscriptions and they much prefer to pay a one time fee, but unfortunately that is not an option. So please take this into account if you answer the question.
- unrelated to the **pro** questions, but is anyone still using ftp/ftps syncing protocols? Do you still find them useful? I'm thinking of deprecating them and eventually removing these two features.   

From my experience there is a lot of backlash when something turns suddenly in a paid version. I want to stress out that this is not the case. Sourcesync as a free version will still go on, as a base for the **pro** version. It will still receive bug fixes, and new features.

Thank you for your time, for your feedback and for your contributions to Sourcesync. 

Edit:
My sincere apologies to everyone notified about this subject but not interested in it. I don't have any other way of determining who is an active user. Please use the unsubscribe button to disable any further notifications.
![image](https://user-images.githubusercontent.com/1479167/211154749-e5e5218b-472d-4f2c-8b3c-7a1e006fc459.png)

**Comment by AllenJB** [comment]
Spam mentions? That's a dick move.

Even if I still used this plugin, I wouldn't pay someone who pulls this.

Edit: lol - you haven't even fixed the bug I commented on 5 years ago.

**Reply by fioan89** [reply]
I'm terribly sorry for any inconvenience I have created. It's an honest discussion in which I'm interested in the feedback from the community. Is it wrong to ask the users for feedback? There is no need to be aggressive. You can simply ignore the discussion if it's too bothersome.

WRT to the issue you mentioned: SCP as a protocol is not able (I think this is documented in the README) to create directories when synchronizing files. It can create a remote directory only when you sync a local directory., This feature is under discussion now. For SSH those issues reported were all fixed, but never got the chance to clean the backlog.

**Reply by apoloval** [reply]
No, this is not a honest discussion. As you have pushed the people into the conversation dishonestly. This is pure spam.

**Reply by apoloval** [reply]
BTW, rest of people can [report this spam using this form](https://support.github.com/contact/report-abuse?category=report-abuse&report=fioan89&report_id=1479167&report_type=user#contact_comments).

**Comment by DrLuke** [comment]
If you see a way to successfully monetize your work you absolutely should, and more power to you!

But please don't randomly tag a random bunch of people, I have no idea why you tagged me or what this project even is.

**Reply by fioan89** [reply]
That is odd, you have contributed with feedback on this issue:  https://github.com/fioan89/sourcesync/issues/14#issuecomment-207659466

**Reply by DrLuke** [reply]
That was like 6 years ago

**Reply by fioan89** [reply]
Yes it is.  I don't know who is still actively using the product. I just grokked all the issues since 2015 and compiled a list of users who contributed in one way or another to the plugin. No harm intended.

**Reply by DrLuke** [reply]
I believe you that harm wasn't intended, but please don't do it again 🙏

**Comment by Cmoen11** [comment]
How did you even get my username? I have to my knowledge never used this. I've reported this as spam. You're no different than the spam emails in my inbox. Luckily Gmail catches them and moves them to my junk folder, github should catch up.

**Reply by fioan89** [reply]
Weird, was your account breached? Because you voted on synchronizing entire directories: https://github.com/fioan89/sourcesync/issues/16#issuecomment-306005366

**Comment by fioan89** [comment]
Everyone that is not interested in this discussion, please use the unsubscribe button from the top right panel. Reporting this as spam will harm users that might still have an active interest in the plugin and the future roadmap.  

My sincere apologies to everyone notified about this subject but not interested in it. I don't have any other way of determining who is an active user.
![image](https://user-images.githubusercontent.com/1479167/211154587-58ee44ab-c4b8-4829-a9d3-caf037e6336e.png)
